sample/src/main/java/org/smartregister/nativeform/interactor/FormTesterInteractor.java

```diff
@@ -0,0 +1,185 @@
+package org.smartregister.nativeform.interactor;
+
+import android.content.Context;
+import android.content.res.AssetManager;
+import android.os.Environment;
+import android.os.Handler;
+import android.os.Looper;
+import android.support.annotation.Nullable;
+
+import com.google.gson.Gson;
+import com.vijay.jsonwizard.constants.JsonFormConstants;
+import com.vijay.jsonwizard.domain.Form;
+
+import org.jetbrains.annotations.NotNull;
+import org.smartregister.nativeform.contract.FormTesterContract;
+import org.smartregister.nativeform.domain.ConfigForm;
+import org.smartregister.nativeform.domain.JsonForm;
+import org.smartregister.nativeform.util.FileReaderUtil;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class FormTesterInteractor implements FormTesterContract.Interactor {
+    private Gson gson = null;
+    private ExecutorService executorService = Executors.newFixedThreadPool(1);
+
+    @Override
+    public void exportDefaultForms(Context context, FormTesterContract.Presenter presenter) {
+        executorService.execute(() -> {
+            AssetManager assetManager = context.getAssets();
+            try {
+                String root = verifyOrCreateDiskDirectory();
+
+                String[] image_source = {"json.form", "json.form.config", "img", "image", "rule"};
```

Comment:
`json.form` this appears throughout the code, maybe use a constant in it's place?

Reply:
Also, snake case? :)